charts/latest-v2/azuredisk-csi-driver/values.yaml

```diff
@@ -74,6 +74,9 @@ controller:
   provisionerWorkerThreads: 40
   attacherWorkerThreads: 500
   vmssCacheTTLInSeconds: -1
+  enableAzureClientAttachDetachRateLimiter: false
```

Comment:
Are we going to disable attach detach rate limiter by default?

Reply:
This is for standalone runs in Aurora. I'll check if there is a way to configure this in Aurora instead of here.